registration/src/test/java/org/spine3/samples/lobby/registration/seat/availability/testcase/ExistAvailableSeatsAndPendingReservations.java

```diff
@@ -53,13 +53,13 @@
             newSeatQuantity(220));
 
     @Override
-    public TestSeatsAvailabilityAggregate givenAggregate() {
-        final TestSeatsAvailabilityAggregate aggregate = super.givenAggregate();
+    public SeatsAvailabilityAggregate givenAggregate() {
+        final SeatsAvailabilityAggregate aggregate = super.givenAggregate();
         final SeatsAvailability state = aggregate.getState().toBuilder()
                 .addAllAvailableSeat(AVAILABLE_SEATS)
                 .putAllPendingReservations(PENDING_RESERVATIONS)
                 .build();
-        aggregate.incrementState(state);
+        aggregate.testIncrementState(state);
```

Comment:
Please rename 
- `testIncrementState()` to `incrementStateForTest()`;
- `testDispatch()` to `dispatchForTest()`.

This way they 1) would sort better with the production counterparts; 2) won't be confused with testing methods.